cfme/containers/provider/openshift.py

```diff
@@ -48,10 +48,39 @@ def view_value_mapping(self):
             'hostname': self.hostname,
             'api_port': self.api_port
         }
-        if current_version() >= '5.8':
-            out['sec_protocol'] = self.sec_protocol
-            if self.sec_protocol.lower() == 'ssl trusting custom ca':
-                out['trusted_ca_certificates'] = OpenshiftDefaultEndpoint.get_ca_cert()
+
+        out['sec_protocol'] = version.pick({
+            version.LOWEST: None,
+            '5.8': self.sec_protocol})
```

Comment:
**REQUIRED**
5.7 is not supported anymore
```python
out['sec_protocol'] = self.sec_protocol
```

Reply:
Fixed